src/sentry/static/sentry/app/views/settings/project/filtersAndSampling/rules/rule/actions.tsx

```diff
@@ -44,15 +63,39 @@ function Actions({disabled, onEditRule, onDeleteRule}: Props) {
           shouldConfirm={false}
           icon={<IconDownload size="xs" />}
           title={t('Edit')}
+          onClick={
+            !disabled
+              ? onEditRule
+              : event => {
+                  event?.stopPropagation();
+                }
+          }
```

Comment:
What happens if this is not here and the element is just disabled?

Reply:
the dropdown menu will close